rapid7_insightvm/plugin.spec.yaml

```diff
@@ -2182,6 +2182,18 @@ actions:
         description: 'Tag search criteria - options documentation: https://help.rapid7.com/insightvm/en-us/api/#section/Responses/SearchCriteria'
         type: object
         required: true
+      size:
+        title: Size
+        description: The number of records to retrieve if blank or 0 all assets that match the search will be returned
+        type: number
+        required: false
+        example: 100
+      sort:
+        title: Sort
+        description: The criteria to sort the records by, in the format property[,ASC|DESC]. The default sort order is ascending. Multiple sort criteria can be specified using multiple sort query parameters.
```

Comment:
Can you simplify this for the user in any way?

Reply:
a thought, maybe we could split this into two inputs because this syntax is confusing.

```
order:
  type: string
  enum:
    - Ascending
    - Descending
  default: ascending
field:
  type: string
```